app/src/main/java/info/touchimage/demo/TouchSyncActivity.kt

```diff
@@ -0,0 +1,54 @@
+package info.touchimage.demo
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import com.ortiz.touchview.OnTouchImageViewListener
+import info.touchimage.demo.databinding.ActivityTouchSyncBinding
+
+class TouchSyncActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityTouchSyncBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityTouchSyncBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
+
+        binding.touchImageOverlay.setOnTouchImageViewListener(object : OnTouchImageViewListener {
+            override fun onMove() {
+                binding.touchImage1.setZoom(binding.touchImageOverlay)
+                binding.touchImage2.setZoom(binding.touchImageOverlay)
+            }
+        })
+
+        // Works
+        val img = getBitmap(this, R.drawable.ic_baseline_aspect_ratio_24)
```

Comment:
@doplerfish @phillies
I tried out the 'Works' section of the code, and in my point of view, this doesn't work either. Or I simply don't understand the issue.
`ic_baseline_aspect_ratio_24` is not in sync in move during zoom.
https://user-images.githubusercontent.com/3314607/174240655-c737c643-a50c-46fa-a8fe-1e1d6c3c7fd3.mp4

Btw, I do what I can and I keep this repository alive by merging pull requests, but I don't fix any issue of other guys. You can help me too !